ComputeClient/Compute.Client.UnitTests/ApiUriTests.cs

```diff
@@ -15,15 +15,15 @@ public void ReturnsGetDomainNatRulesUri()
         {
             var networkDomainId = Guid.NewGuid();
             var uri = ApiUris.GetDomainNatRules(_orgId, networkDomainId.ToString());
-            Assert.AreEqual(ApiUris.MCP2_9_PREFIX + _orgId + "/network/natRule?networkDomainId="+networkDomainId, uri.OriginalString);
+            Assert.AreEqual(ApiUris.MCP2_9_PREFIX + _orgId + "/network/natRule?networkDomainId=" + networkDomainId, uri.OriginalString);
```

Comment:
formated file using Visual Studio 2017 - real changes in this file start on line 158